### PR TITLE
fix website cdn for asset, widget image, widget html

### PR DESCRIPTION
**Pull request by Gorash**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Gorash** [comment]
reviewed ?

**Comment by antonylesuisse** [comment]
Nope not good to me.

File: addons/website/models/website.py [hunk 1]
```diff
@@ -563,15 +567,13 @@ def image_url(self, record, field, size=None):
...
-    @api.model
     def get_cdn_url(self, uri):
-        # Currently only usable in a website_enable request context
-        if request and request.website and not request.debug and request.website.user_id.id == request.uid:
-            cdn_url = request.website.cdn_url
-            cdn_filters = (request.website.cdn_filters or '').splitlines()
-            for flt in cdn_filters:
-                if flt and re.match(flt, uri):
-                    return urls.url_join(cdn_url, uri)
+        self.ensure_one()
```
Comment (Gorash) [inline addons/website/models/website.py]: if not uri: return ''

**Comment by rdeodoo** [comment]
This was merged by @JKE-be

**Pull request closed without merging**